File: 3074-apple-redistribution-into-boxes/3074-apple-redistribution-into-boxes.ts
function minimumBoxes(apple: number[], capacity: number[]): number {
        let tot = apple.reduce((a, b) => a + b);
    capacity.sort((a, b) => b - a);
let hello =0

while(tot>0){
    tot-=capacity[hello++];
}
    return hello;
};

File: 3074-apple-redistribution-into-boxes/3074-apple-redistribution-into-boxes.cs
public class Solution
{
    public int MinimumBoxes(int[] apple, int[] capacity)
    {
        int tot = 0;
        foreach (int a in apple)
            tot += a;

        Array.Sort(capacity, (a, b) => b - a);

        int hello = 0;

        while (tot > 0)
        {
            tot -= capacity[hello++];
        }

        return hello;
    }
}
